minutes/2025/BoD-minutes-Q2-2025-05-22.md

```diff
@@ -0,0 +1,166 @@
+# Board of Directors Meeting Minutes for Q2 2025-05-22
+
+## **Present on 2025-05-22**: 
+
+Samantha, Erin, Yani, Pao, Jannetta, Kari, Stephanie, Luke, Mark, Nisha, Yo, Danielle, Eboni
+
```

Comment:
Phil arrive late, but he was at the meeting, don't remember exactly how much time.

Reply:
thanks, added in!